packages/app/package.json

```diff
@@ -11,9 +11,10 @@
   },
   "dependencies": {
     "@feathersjs/adapter-commons": "4.5.11",
+    "@feathersjs/authentication-client": "4.5.11",
     "@feathersjs/feathers": "4.5.11",
     "@feathersjs/socketio-client": "4.5.11",
-    "@feathersjs/authentication-client": "4.5.11",
+    "@fortawesome/fontawesome-free": "^5.15.1",
```

Comment:
we should communicate that we use another font in the next meeting, so this font is used unified, else we will have different fonts.
Or maybe enforce this font for everyone in some way

Reply:
we do not change the font in this branch. We use the fortawesome plugin to show icons in the textfield.

Reply:
oh okay